uaclient/contract.py

```diff
@@ -93,8 +94,11 @@ def add_contract_machine(
                 code=response.code,
                 body=response.body,
             )
-
-        return response.json_dict
+        response_json = response.json_dict
+        secret_manager.secrets.add_secret(
+            response_json.get("machineToken", "")
+        )
+        return response_json
```

Comment:
We should also redact the resourceTokens (same as in machine_token_file.read)
```suggestion
        for token in response_json.get("resourceTokens", []):
            secret_manager.secrets.add_secret(token.get("token", ""))
        return response_json
```